fuel-moshi/src/test/kotlin/com/github/kittinunf/fuel/FuelMoshiTest.kt

```diff
@@ -29,7 +29,7 @@ class FuelMoshiTest {
     @Test
     fun moshiTestResponseObject() {
         Fuel.get("/user-agent")
-                .responseObject(moshiDeserializerOf<HttpBinUserAgentModel>()) { _, _, result ->
+                .responseObject(moshiDeserializerOf(HttpBinUserAgentModel::class)) { _, _, result ->
```

Comment:
I feel like `moshiDeserializerOf<HttpBinUserAgentModel>()` triumphs `moshiDeserializerOf(HttpBinUserAgentModel::class)` (at very least, it is less keystroke). What do you think about it?

Reply:
This is one of the Trade-Off from fixing the moshiDeserializerOf() function.

Reply:
OK understood